moto/appsync/responses.py

```diff
@@ -192,3 +192,60 @@ def get_introspection_schema(self) -> str:
             format_=format_, include_directives=include_directives
         )
         return schema
+
+    def get_api_cache(self) -> str:
+        api_id = self.path.split("/")[-2]
+        api_cache = self.appsync_backend.get_api_cache(
+            api_id=api_id,
+        )
+        return json.dumps(dict(apiCache=api_cache.to_json()))
+
+    def delete_api_cache(self) -> str:
+        api_id = self.path.split("/")[-2]
+        self.appsync_backend.delete_api_cache(
+            api_id=api_id,
+        )
+        return "{}"
+
+    def create_api_cache(self) -> str:
+        params = json.loads(self.body)
+        api_id = self.path.split("/")[-2]
```

Comment:
I *think* you can get this var with `self._get_param("appId")`

Reply:
Yeah I saw that function elsewhere, but it doesn't seem to work in this case.  Not quite sure why